policyfile.go

```diff
@@ -159,6 +162,15 @@ type NodeAttrGrantApp struct {
 	Domains    []string `json:"domains,omitempty" hujson:"Domains,omitempty"`
 }
 
+// ACLAttrConfig represents configuration for a custom device attribute.
+type ACLAttrConfig struct {
+	Type string `json:"type,omitempty" hujson:"Type,omitempty"`
```

Comment:
Maybe worth enumerating the valid types? I think this is correct as it's based on posture attributes right?

```suggestion
	// Type can be one of "string", "bool", or "number"
	Type string `json:"type,omitempty" hujson:"Type,omitempty"`
```